public/controllers/agent/components/register-agent.js

```diff
@@ -115,34 +115,33 @@ const pTextCheckConnectionStyle = {
   marginTop: '3em',
 };
 
-export const RegisterAgent = withErrorBoundary(
```

Comment:
**thought**: Why the `withErrorBoundary` was removed?

Reply:
I haven't seen it when I resolved conflicts. Thanks for reporting!